extension/src/lib/zoom.js

```diff
@@ -0,0 +1,50 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+fluid.defaults("gpii.chrome.zoom", {
+    gradeNames: "fluid.modelComponent",
+    model: {
+        magnifierEnabled: false,
+        magnification: 1
+    },
+    invokers: {
+        set: {
+            funcName: "gpii.chrome.zoom.set",
+            args: "{arguments}.0"
+        }
+    },
+    modelListeners: {
+        "*": {
```

Comment:
Should be "" rather than "*" - please see docs: http://docs.fluidproject.org/infusion/development/ChangeApplierAPI.html#wildcards-in-model-path-references